source/to_cpp1.h

```diff
@@ -4882,9 +4882,9 @@ class cppfront
         function_type_node const& n,
         bool                      is_main                    = false,
         bool                      is_ctor_or_dtor            = false,
-        std::string               suffix1                    = {},
+        std::string const&        suffix1                    = {},
```

Comment:
Default value for a const ref argument always made me cringe, but again, this might be personal taste 😉.
Nevertheless, wouldn't `std::string_view` passed by value work instead?

Reply:
definitely better with a std::string_view, I was just trying to do minimal cleanup with minimal changes

There's a lot more places in the code where a std::string_view could replace a string, it's on my TODO list... for both constness and performance reasons.

Reply:
will have them in a follow-up pass, I rebased the whole thing on top of current master, with the suggested changes.